runtime/gc_glue_java/configure_includes/configure_linux_arm.mk

```diff
@@ -38,21 +44,29 @@ ifeq (linux_arm, $(SPEC))
 		--enable-OMR_PORT_CAN_RESERVE_SPECIFIC_ADDRESS
 endif
 
-# Override the GLOBAL_INCLUDES value provided by configure_common.mk
-CONFIGURE_ARGS += 'GLOBAL_INCLUDES=/bluebird/tools/arm/arm-bcm2708/arm-bcm2708hardfp-linux-gnueabi/arm-bcm2708hardfp-linux-gnueabi/include' 
+ifdef OPENJ9_CC_PREFIX
+  CONFIGURE_ARGS += 'AS=$(OPENJ9_CC_PREFIX)-as'
+  CONFIGURE_ARGS += 'CC=$(OPENJ9_CC_PREFIX)-gcc'
+  CONFIGURE_ARGS += 'CXX=$(OPENJ9_CC_PREFIX)-c++'
+  CONFIGURE_ARGS += 'AR=$(OPENJ9_CC_PREFIX)-ar'
+  CONFIGURE_ARGS += 'OBJCOPY=$(OPENJ9_CC_PREFIX)-objcopy'
+else
+  # Override the GLOBAL_INCLUDES value provided by configure_common.mk
+  CONFIGURE_ARGS += 'GLOBAL_INCLUDES=/bluebird/tools/arm/arm-bcm2708/arm-bcm2708hardfp-linux-gnueabi/arm-bcm2708hardfp-linux-gnueabi/include'
+  CONFIGURE_ARGS += 'AS=bcm2708hardfp-as'
```

Comment:
Perhaps it might make more sense to provide a default for `OPENJ9_CC_PREFIX` (bcm2708hardfp) so `AS`, `CC`, etc. can be added to configure args just one way each.

Reply:
Yes, I'd much prefer the approach you suggest too. I was trying to be ultra cautious about breaking existing behaviour, but I think the benefits of your suggestion outweigh the risks of getting it wrong.